test/fuzz/syntax.ml

```diff
@@ -0,0 +1,5 @@
+let ( let* ) = Crowbarplus.dynamic_bind
+
+let ( let+ ) gen map_fn = Crowbarplus.map [ gen ] map_fn
+
+let ( and+ ) = Crowbarplus.pair
```

Comment:
We can also remove this one (you'll have to clone crowbar and run `dune install` until it's available on opam).